Microsoft.Toolkit.Uwp.UI.Controls/Menu/Menu.Logic.cs

```diff
@@ -22,7 +22,8 @@ public partial class Menu
         private const string CtrlValue = "CTRL";
         private const string ShiftValue = "SHIFT";
         private const string AltValue = "ALT";
-        private FlyoutPlacementMode? _currentFlyoutPlacement;
+
+        internal FlyoutPlacementMode? CurrentFlyoutPlacement { get; set; }
```

Comment:
Add a comment about why this is internal now (i.e. the child menu item needs it to respect the menu direction of submenus)?

Reply:
done